### PR TITLE
The most important update

### PR DESCRIPTION
**Pull request by Ratismal**
Hey there, LoganDark! 

I see you've made some pretty big changes to my bot, blargbot, and I have to say that I approve! You've made the bot over 9000% more efficient, at the very least. I might consider using some of these changes in my production version!

However, your branch was last updated 5 months ago. In this time, I have pushed an update to make the bot run even smoother. I thought I would PR it into your branch to help your development.

I'm glad you're enjoying my work! I look forwards to our next collaboration ;3

Sincerely
ratismal (stupid cat)

### CODE REVIEW

**Comment by SnazzyPine25** [comment]
Hey dude! I heard you like spamming! So I put spam in your spamming!

File: blargbot.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Now look at this net,
+// that I just found!
+async function net() {
```
Comment (TTtie) [inline blargbot.js]: ~~use sync~~

**Comment by SnazzyPine25** [comment]
@Ratismal Add cage

**Comment by Ratismal** [comment]
Hmm, good point. I shouldn't assume a harmony workspace.

**Comment by LoganDark** [comment]
i   a p p r o v e

**Comment by SnazzyPine25** [comment]
get online nibba

**Comment by LoganDark** [comment]
👀

**Comment by SnazzyPine25** [comment]
cmon boi

**Comment by SnazzyPine25** [comment]
👀 👀 👀 👀

**Comment by Ratismal** [comment]
Thanks! I look forwards to our next collaboration!

**Pull request merged**